app/components/EventItem/styles.css

```diff
@@ -6,6 +6,13 @@
   display: flex;
   justify-content: space-between;
   line-height: 1.3;
+  transition: var(--easing-fast);
+}
+
+.eventItemHover {
```

Comment:
```suggestion
.eventItem:hover {
```

You can do this instead of using TypeScript!

Reply:
➕